src/core/server/index.ts

```diff
@@ -288,7 +288,15 @@ export {
   MetricsServiceSetup,
 } from './metrics';
 
-export { RecursiveReadonly } from '../utils';
+export {
+  RecursiveReadonly,
+  DEFAULT_APP_CATEGORIES,
```

Comment:
@joshdover I'm surprised we have this in utils. Shouldn't we move it to the `application` or another service instead?  `utils` are meant to provide functionality extending the language standard library.

Reply:
I believe this is utils due to problems with importing from server into public. We can't use the server/types workaround since this is a value. Long-term we could fix this by either leveraging a common directory or the organize by domain proposal for core #52182

Reply:
Ok, I think we can get rid of the server-side use-cases when ~all plugin migrate to KP~ most plugins migrate to KP and we can inline value in the remaining cases.

Reply:
`DEFAULT_APP_CATEGORIES` 'util' was introduced during the grouped nav PR, because these values needed to be accessed by the legacy apps registrations that are performed on the server-side. I agree with @restrry that once we only have a small amount of legacy app registrations, moving this back to client-side and using inline values for the legacy apps would be an acceptable solution.